### ISSUE TITLE
CycleGAN convergence

### ISSUE THREAD

**Issue by wjx2** [junyanz/pytorch-CycleGAN-and-pix2pix#335]
Hi . Thanks for the wonderful explanation. I am new in GAN. And I have a question.
When I train cycleGAN, I found that the discriminator loss decreases and converges, but the generator loss always change not much. So, How to judge whether cycleGAN converges?

**Comment by junyanz** [comment]
Many GAN losses do not converge at all (exception: WGAN) due to the nature of minimax optimization. Your loss curves look quite normal.

**Comment by ajaykmr2292** [comment]
Can you tell me what does GAN loss exploding signify and when does that happen?

**Comment by junyanz** [comment]
Sometimes GAN loss might explode if you have an unusual training input (e.g., a white image). But it should be fine as long as the loss is reasonable for most of the times.
